### PR TITLE
preprocessing html report

### PR DESCRIPTION
**Pull request by kalenkovich**

### CODE REVIEW

File: 01_preprocessing/report.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+from pathlib import Path
+import os
+
+
+from IPython.display import Image
+import pandas as pd
+
+
+import mne
```
Comment (kalenkovich) [inline 01_preprocessing/report.py:12]: Че за mnе-то? Давай spm.

File: conda-environment.yml [hunk 2]
```diff
@@ -5,7 +5,8 @@ dependencies:
   - ipykernel=5.1.4
   - jupyter
   - conda-forge::mne=0.20.5
-  - conda-forge::mne-bids
+  - conda-forge::mne-bids=2.2.2
```
Comment (levchenkoegor) [inline conda-environment.yml:8]: Looks like mne-bids 2.2.2 doesn't exist, the latest version is 0.6. Should it be changed or I misunderstand someting?
We haven' specified mne-bids version in previous commits, I suggest 0.5 version.

File: conda-environment.yml [hunk 3]
```diff
@@ -5,7 +5,8 @@ dependencies:
...
   - bioconda::snakemake-minimal=5.3.0
+  - conda-forge::papermill=5.3.1
```
Comment (levchenkoegor) [inline conda-environment.yml:10]: The latest version of papermill 2.2.2?

Comment (kalenkovich) [reply conda-environment.yml:10]: You're right. Will fix.

File: conda-environment.yml (hunk 2 above)
Comment (kalenkovich) [reply conda-environment.yml:8]: Both the server-wide environment you use and my environment use 0.4. Sheer luck the versions are the same. I'll set the version together with the correct papermill version.

**Review by levchenkoegor — CHANGES_REQUESTED** [review]
Need to rebase onto master

**Comment by kalenkovich** [comment]
Rebased onto master. No new changes introduced.

**Pull request merged**